packages/in_app_purchase/lib/src/store_kit_wrappers/sk_receipt_manager.dart

```diff
@@ -14,8 +14,12 @@ class SKReceiptManager {
   /// There are 2 ways to do so. Either validate locally or validate with App Store.
   /// For more details on how to validate the receipt data, you can refer to Apple's document about [`About Receipt Validation`](https://developer.apple.com/library/archive/releasenotes/General/ValidateAppStoreReceipt/Introduction.html#//apple_ref/doc/uid/TP40010573-CH105-SW1).
   /// If the receipt is invalid or missing, you can use [SKRequestMaker.startRefreshReceiptRequest] to request a new receipt.
-  static Future<String> retrieveReceiptData() {
-    return channel.invokeMethod<String>(
-        '-[InAppPurchasePlugin retrieveReceiptData:result:]');
+  static Future<String> retrieveReceiptData() async {
```

Comment:
Could this return null before? Wondering whether or not this return type change is going to potentially cause NPEs to anyone who wasn't getting them before.

Reply:
It could potentially return null before:
https://github.com/flutter/plugins/blob/master/packages/in_app_purchase/ios/Classes/FIAPReceiptManager.m#L24